index.js

```diff
@@ -1,7 +1,10 @@
 const { S3Client, GetObjectCommand, PutObjectCommand } = require('@aws-sdk/client-s3')
 const { MultihashIndexSortedWriter } = require('cardex')
 const { CarIndexer } = require('@ipld/car/indexer')
-const { concat } = require('uint8arrays')
+const { Readable } = require('stream')
+
+const maxRetries = process.env.MAX_RETRIES ? parseInt(process.env.MAX_RETRIES) : 3
+const retryDelay = process.env.RETRY_DELAY ? parseInt(process.env.RETRY_DELAY) : 500
```

Comment:
the default we set on indexer is 100 ms, we've seen 500 is too much - the lambda execution cost would be too expensive